tools/integration-tests/tester/framework/framework.go

```diff
@@ -84,3 +90,75 @@ func (f *Framework) CreateNetwork(name string, peers int, minimumNeighbors int)
 
 	return network, nil
 }
+
+// CreateDRNGNetwork creates and returns a (Docker) Network that contains drand and `peers` GoShimmer nodes.
+func (f *Framework) CreateDRNGNetwork(name string, members, peers int) (*DRNGNetwork, error) {
+	drng, err := newDRNGNetwork(f.dockerClient, strings.ToLower(name), f.tester)
+	if err != nil {
+		return nil, err
+	}
+
+	err = drng.network.createEntryNode()
+	if err != nil {
+		return nil, err
+	}
+
+	// create members/drand nodes
+	for i := 0; i < members; i++ {
+		leader := i == 0
+		if _, err = drng.CreateMember(leader); err != nil {
+			return nil, err
+		}
+	}
+
+	// wait until containers are fully started
+	time.Sleep(60 * time.Second)
+	err = drng.WaitForDKG()
+	if err != nil {
+		return nil, err
+	}
+
+	// create GoShimmer identities
+	pubKeys := make([]hive_ed25519.PublicKey, peers)
+	privKeys := make([]hive_ed25519.PrivateKey, peers)
+	var drngCommittee string
+
+	for i := 0; i < peers; i++ {
+		pubKeys[i], privKeys[i], err = hive_ed25519.GenerateKey()
+		if err != nil {
+			return nil, err
+		}
+
+		if i < members {
+			if drngCommittee != "" {
+				drngCommittee += fmt.Sprintf(",")
+			}
+			drngCommittee += fmt.Sprintf("%s", base58.Encode(pubKeys[i][:]))
+		}
+	}
+
+	conf := GoShimmerConfig{
+		DRNGInstance:  1,
+		DRNGThreshold: 3,
+		DRNGDistKey:   hex.EncodeToString(drng.distKey),
+		DRNGCommittee: drngCommittee,
+	}
+
+	// create peers/GoShimmer nodes
+	for i := 0; i < peers; i++ {
+		conf.Bootstrap = i == 0
+		conf.Seed = base64.StdEncoding.EncodeToString(ed25519.PrivateKey(privKeys[i].Bytes()).Seed())
+		if _, err = drng.CreatePeer(conf, pubKeys[i]); err != nil {
+			return nil, err
+		}
+	}
+
+	// wait until peers are fully started and connected
+	time.Sleep(1 * time.Second)
+	err = drng.network.WaitForAutopeering(3)
```

Comment:
Why set `minimumNeighbors` here hardcoded? I think it should be a parameter to the function